aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -1032,12 +1047,14 @@ module AptosFramework::Stake {
         );
     }
 
-    public fun is_validator_active(pool_address: address): bool acquires ValidatorSet {
+    /// Return true if the current validator is not active or pending_active in the current epoch.
+    /// This still includes pending_inactive validators as they are not deactivated until the next epoch and can still
```

Comment:
this comment doesn't describe what the code does, it doesn't include pending_inactive, not sure which one you wanted

Reply:
Updated comment